appveyor.yml

```diff
@@ -46,7 +49,6 @@ test_script:
   - bundler --version
   - bash ./script/cibuild
 
-cache:
+#cache:
   # If one of the files after the right arrow changes, cache will be skipped
-  - C:\Ruby%RUBY_FOLDER_VER%\bin -> Gemfile,jekyll.gemspec,appveyor.yml
-  - C:\Ruby%RUBY_FOLDER_VER%\lib\ruby\gems\%GEMS_FOLDER_VER% -> Gemfile,jekyll.gemspec,appveyor.yml
+#  - 'vendor -> appveyor.yml,Gemfile,jekyll.gemspec'
```

Comment:
I can go back to the previous solution but to me caching the whole bin folder seems bad. Or the system wide gems folder...